localtileserver/widgets.py

```diff
@@ -196,7 +197,7 @@ def get_folium_tile_layer(
         )
 
     class FoliumTileLayer(TileLayer, LocalTileServerLayerMixin):
-        pass
+        bounds = Union((Tuple(),), default_value=None, allow_none=True).tag(sync=True, o=True)
```

Comment:
I don't think this is needed, is it? The `ipyleaflet` implementation does this because `ipyleaflet` provides `ipywidgets` and the this is required to sync the bounds to the JS object. My understanding is tat folium is entirely different and just passing the `bounds` as a keyword argument to `folium.TileLayer` is sufficient. I'll check this out locally though and verify whats needed

Reply:
Yep! I confirmed this locally. Going to push up the modifications to your branch